gtdynamics/utils/Phase.cpp

```diff
@@ -26,7 +26,7 @@ using std::string;
 std::ostream &operator<<(std::ostream &os, const Phase &phase) {
   os << "[";
   for (auto &&cp : phase.contactPoints()) {
-    os << cp.first << ": " << cp.second << ", ";
+    os << cp.link->name() << ": " << cp.point << ", ";
```

Comment:
@jerredchen Try fixing test by changing to point.transpose() ? Feel free to directly add this branch now.

Reply:
Sounds good, will do.

Reply:
@dellaert Just updated it, and it's passing the unit test now. Thanks!